cloudevents/tests/test_pydantic_event_attribute_types.py

```diff
@@ -0,0 +1,273 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cloudevents.pydantic.event_attribute_types import (
+    AbsoluteURI,
+    ContentType,
+    NonEmptyAbsoluteURI,
+    NonEmptyURIReference,
+    URIReference,
+)
+
+ABSOLUTE_URIS = [
```

Comment:
That's actually what I'd like to avoid. It's not up to the SDK or its part to perform such validations IMO. I'd rather make the URI type less restrictive.

Reply:
I have created a strict cloudevent whic enforces the spec. And this type is only used there